docs/troubleshooting.md

```diff
@@ -89,6 +89,13 @@ xorg-x11-fonts-misc
 - read about linux sandbox here: https://chromium.googlesource.com/chromium/src/+/master/docs/linux_suid_sandbox_development.md
 - try running without the sandbox (**Note: running without the sandbox is not recommended due to security reasons!**)
 
+```js
+const browser = await puppeteer.launch({args: ['--no-sandbox', '--disable-setuid-sandbox']});
+```
+## Running Puppeteer on Travis CI
+
+To run headless Chrome on Travis, you *must* call `launch()` with flags to disable Chrome's sandbox, like so:
+
```

Comment:
nit: the `--disable-setuid-sandbox` is not needed.

https://github.com/GoogleChrome/puppeteer/blob/717d4a7ada6f82a8dce8f801a7136d15c3b17df1/test/test.js#L44-L49